db/seeds.rb

```diff
@@ -59,14 +59,19 @@
 puts "---> Responding to some consultations"
 Consultation.where(status: [:published, :rejected]).each do |consultation|
 	response_count = [0, 1, 2, 3, 4, 5].sample
-	Fabricate.times(response_count, :consultation_response, consultation_id: consultation.id)
+	Fabricate.times(response_count, :response_round, consultation_id: consultation.id)
+	begin
+		puts "---> Creating Response Round for consultations"
+		Fabricate.times(response_count, :consultation_response, consultation_id: consultation.id, response_round_id: consultation.response_rounds.last.id)
+	rescue
```

Comment:
@BALAJI-BALA - we should never just "rescue" bro, I suggest at least printing out the error.